modules/service/data-service.js

```diff
@@ -8,6 +8,7 @@ class DataService {
         this.logger = ctx.logger;
         this.constants = ctx.constants;
         this.validationService = ctx.validationService;
+        this.networkService = ctx.networkService;
```

Comment:
Communication between services is not recommended, it should be independent and retrieve all parameters from config or ot-node

Reply:
Let's leave this discussion for later, as we have more details of the update of architecture that we are preparing